src/app/resource/resource-list/resource-list.component.html

```diff
@@ -10,7 +10,14 @@ <h6 class="card-subtitle text-muted">Resource</h6>
         <h6 class="card-subtitle text-muted">Description</h6>
         <p class="card-text">{{resource.description}}</p>
       </div>
-      <!-- Rest of attributes-->
+      <div class="col">
+        <h6 class="card-subtitle text-muted">Resource type</h6>
+        <p class="card-text">{{resource.resourceType}}</p>
+      </div>
+      <div class="col">
+        <h6 class="card-subtitle text-muted">Owner</h6>
+        <p class="card-text">{{resource.owner.username}}</p>
```

Comment:
View fails because the owner object is not retrieved and the username is undefined. You should retrieve owners by following the corresponding relation, like in: https://github.com/UdL-EPS-SoftArch/UniCloud/blob/main/src/app/rating/rating-list/rating-list.component.ts#L35